includes/class-alg-wc-custom-order-numbers-core.php

```diff
@@ -586,7 +586,7 @@ public function alg_wc_webhook_after_cutoff_con() {
 						$order_con = true;
 					}
 					if ( $order_con ) {
-						update_post_meta( $order_id, '_alg_wc_full_custom_order_number', $full_order_number );
+						$order->update_meta_data( '_alg_wc_full_custom_order_number', $full_order_number );
```

Comment:
Whenever you add update_meta_data(), you will have to add $order->save() to save the values. Please test it once.